src/Passes/QirExamples/ParallelQRng/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples {
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Intrinsic;
+
+    operation T(q: Qubit): Int
+    {
```

Comment:
Is there a significance to the name `T` here? That's also the name of a commonly used quantum gate, such that it's a little confusing for `T` to be reused for a test no-op.
```suggestion
    operation T(q : Qubit) : Int  {
```